contractor/processing_chain.cpp

```diff
@@ -100,13 +115,46 @@ int Prepare::Run()
     return 0;
 }
 
+namespace std {
+
```

Comment:
Can you reformat this section using the `.clangformat` file in the repository root?